config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -377,7 +374,7 @@ periodics:
     preset-kind-volume-mounts: "true"
   decorate: true
   decoration_config:
-    timeout: 200m
```

Comment:
same question above - Did you mean to change the timeouts?

Reply:
yes